bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -612,12 +612,16 @@ class ProblemRobot(s: ShellAdvisor) extends Robot {
       }
 
     private val noPIPhoneNumber = when (p.investigators.pi.phone.isEmpty) {
-      new Problem(Severity.Warning, s"No phone number given for ${investigatorFullName(p.investigators.pi, "PI")}. This is for improved user support.",
-        "Overview", s.inOverview{
-          _.editPi(_.Phone.requestFocus)
-        })
+      new Problem(Severity.Warning,
+        s"No phone number given for ${investigatorFullName(p.investigators.pi, "PI")}. This is for improved user support.",
+        "Overview", s.inOverview(_.editPi(_.Phone.requestFocus)))
     }
 
+    private val invalidPIPhoneNumber = (for {
+      badPhone <- p.investigators.pi.phone if badPhone.count(_.isDigit) < 10
```

Comment:
Actually, looking online, the assumption that a phone number will contain at least 10 digits is wrong. Even with country code, Swedish phone numbers can have nine digits, and phone numbers for the Solomon Islands, eight.

Out of the partner countries, they will all have 10+, but this seems like a poor assumption to make. I'm going to make note on the JIRA task for this.